### DISCUSSION TITLE
1.35.2

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#21140] [General]
Feel free to discuss the [v1.35.2](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.35.2) release here!

**Comment by vjonas** [comment]
Thank you! ♥‿♥

**Comment by Trebor87** [comment]
Many thanks

But still getting an error with TS011F when trying to OTA it. ![Screenshot_20240201-181945.png](https://github.com/Koenkk/zigbee2mqtt/assets/55857416/da144d9a-15bf-4123-8b6b-3a592b6292ff)

Best regards

**Reply by Nerivec** [reply]
There's no firmware image available in Z2M for that combination of specs. Many devices use that combination of image type & manufacturer code (as often with Tuya), so it is further filtered by manufacturer name to ensure it's getting a proper firmware image for your specific device. `_TZ3000_cayepv1a` isn't in the [list](https://github.com/Koenkk/zigbee-OTA/).
If you know of one, you can follow the instructions in the previous link to get support for it in Z2M.

**Comment by Nardol** [comment]
> https://github.com/Koenkk/zigbee2mqtt/issues/20482 Notify systemd for start, stop, watchdog (@chrthi)

It looks the installation documentation is not yet updated, Koenkk/zigbee2mqtt.io#2475 is not merged.
Is it expected? Should we avoid to change SystemD unit?

**Reply by Koenkk** [reply]
Merged the pr!

**Comment by pietrouk** [comment]
Following the update, all my devices have disappeared from the list. They're still working though. How do I get them back?
database.db file has them listed.

I'm on rpi4-64
Core 2024.1.6
Supervisor 2023.12.1
Operating System 11.4
Frontend 20240104.0
Version of add on 1.35.2-1

**Reply by Nerivec** [reply]
I assume you tried restarting Z2M. Did you try a "Reboot system" of HAOS?

**Reply by pietrouk** [reply]
That's right and no luck

**Comment by ivancea** [comment]
After the update, I could still use everything as expected, but after some minutes, I lost connection with all my devices. Both Hass and Z2M worked the first seconds.
I rollbacked, checked everything worked again, re-updated, and checked that it failed again, so looks reproducible at least in my env.

Everything failing with a `no response received (63)`. Example log:
```
Publish 'set' 'state' to 'Luz sala' failed: 'Error: Command 0x001788010b05e90f/11 genLevelCtrl.moveToLevelWithOnOff({"level":2,"transtime":0}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received (63))'
```

Versions:
```
Versión de Zigbee2MQTT
    1.35.2 commit: unknown
Tipo de coordinador
    ConBee2/RaspBee2
Versión de coordinador
    0x26720700
Coordinator IEEE Address
    0x00212effff07ed0d
Versión de la interfaz
    0.6.152
Zigbee-herdsman-converters version
    18.25.1
Zigbee-herdsman version
    0.33.5
```

**Reply by Nerivec** [reply]
I don't know much about the deconz-side of the codebase, but looks like it's not getting a response from the adapter after a basic request (timing out). Did you check the Z2M logs above this, in case some error happened before these failed commands? You'd have to enable [herdsman debug](https://www.zigbee2mqtt.io/guide/usage/debug.html#zigbee-herdsman-debug-logging) to get more precise logs otherwise.
Did you let the network settle after restart? It can take a few moments if lots of things need to happen due to the new version (interviewing for new features & such).
There's also a [config for deconz regarding delays](https://www.zigbee2mqtt.io/guide/configuration/adapter-settings.html#advanced-configuration); not sure what/if you have there?

```yaml
advanced:
  # Optional: Set the adapter delay, only used for Conbee/Raspbee adapters (default 0).
  # In case you are having issues try `200`.
  # For more information see https://github.com/Koenkk/zigbee2mqtt/issues/4884
  adapter_delay: 0
```

**Reply by ivancea** [reply]
Hi! I gave it time, but it broke some time after the restart, not inmediately after. Updates to other versions worked well, as well as rollbacks to the version before this one.
I tried it again to check those logs, and now it worked, and is still working. So I have no idea what could happen (And I think the backup also includes the logs? Not sure, but I don't remember the time anyway to check them).
I'll return here if it happens again. Thanks!

**Reply by cipacda** [reply]
+1 one on this one, this release broke all my HAOS devices

**Reply by TheRealWaldo** [reply]
Having the same issue.  Rolling back to the previous version returns it to normal operation.

**Reply by dc911x** [reply]
Yep same here, also using a conbee II stick.

**Reply by Albruaje** [reply]
Exactly the same for me. Using Phoscon RaspBee II

**Reply by Nerivec** [reply]
Make sure you check if you have devices stuck in "interviewing" (little icon on top of the device's picture in Z2M UI). After large updates that bring new features to your devices, if some indeed get stuck, it can help to push the button on them (should be "sleepy" end devices, on battery), just a push-and-release (not long-push reset). They otherwise fail interviewing (because of that sleepy mode), and that can lead to your network trying requests again and again, and disrupting other devices. (PS: This applies to EZSP, but I'd guess the same applies to deconz.)

Otherwise, if someone has a herdsman debug log, open an issue and post it there (if someone hasn't already).

**Reply by Koenkk** [reply]
This is Conbee specific, let's continue in https://github.com/Koenkk/zigbee2mqtt/issues/21162

**Comment by bergetun** [comment]
This relase breaks the support for Namrom dimmers :(

**Comment by Seger85** [comment]
@Koenkk just wanted to say thank you, what a great piece of software, donation on its way! Sensational and overwhelming!

I have seen a reported error:

IKEA repeaters dead after update to 1.35.2-1
https://github.com/Koenkk/zigbee2mqtt/issues/21147

which is still preventing me from updating. Can anyone else confirm the error and help solve the problem?

Best regards
Seger

**Comment by merlinpimpim** [comment]
Thanks for the update.

1. But I now get this type of message with my adapter: "**Adapter disconnected, stopping**"
(Nabu casa Skyconnect (up to date) / Multiprotocol / EZSP)
Every 6/7 minutes it seems.
![image](https://github.com/Koenkk/zigbee2mqtt/assets/113300765/01aa178f-4be2-4541-9622-021190d60c38)

2. I also have this problem with Lixee: [https://github.com/Koenkk/zigbee2mqtt/issues/21151](url)

3. I have a **NodOn SIN-4-1-20** device showing an OTA firmware update but it doesn't really seem to exist, the update never applies: 
![image](https://github.com/Koenkk/zigbee2mqtt/assets/113300765/5ec57f50-50ce-4600-bb75-66afa3f3a86b)
![image](https://github.com/Koenkk/zigbee2mqtt/assets/113300765/9c753fb2-f86b-4a2a-bc1c-a86073ad5781)

4. And, Is this update now compatible with Silicon Labs Multiprotocol 2.4.4?

5. Edit: The OTAUpdate/deviceMessage' is still there:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/113300765/39e30247-ddee-402b-a58f-3fd5ffac20bf)

Thanks

**Reply by chris-1243** [reply]
I am using firmware NCP 7.4.0.0 on a Sonoff Dongle-E (based on the latest Gecko SDK 4.4.0) on a test network (Z2M 1.35.2 commit: 06b3395) without any issues. Maybe you require an update of the Silabs multi-protocol add-on as well.

Regarding to your NodOn SIN-4-1-20, your problem is related to this issue I opened which is not solved for the moment unfortunately ([#19224](https://github.com/Koenkk/zigbee2mqtt/issues/19224)). Maybe you may contribute and add some new inputs.

**Reply by merlinpimpim** [reply]
https://github.com/Koenkk/zigbee2mqtt/issues/19224: Post fed :)

> Maybe you require an update of the Silabs multi-protocol add-on as well.

So the answer to my question 4 is 'yes'?

**Reply by chris-1243** [reply]
This update (1.35.2) should work on latest silabs firmware (yes). 

If anything goes wrong, feel free to report the bug to this thread [#21011](https://github.com/Koenkk/zigbee2mqtt/discussions/21011).

**Reply by merlinpimpim** [reply]
I migrated SLM to version 2.4.4 and it works with Z2M 1.35.2 😊
However, topics 1, 2 and 3 from my message above persist.
I'm going to mention point 1 in the thread you mentioned.

**Reply by Nerivec** [reply]
Could you open an issue for 1 and post a herdsman debug log with it? I'm looking into it, but I want to make sure what is triggering the reset, I need the herdsman debug for that.
Seems it is recovering properly now though, so that's a good start... 😄 

PS: post a link here once you have, to avoid duplicates.

**Reply by merlinpimpim** [reply]
@Nerivec Thanks for the feedback.
Just a detail: I've just discovered that Z2M crashes and restarts every time the message (Adapter disconnected, stopping) appears in the logs. Also every time I extract the container logs (with docker logs CONTAINER_ID > log.txt 2>&1), I have the Z2M startup sequence and a bit of information but probably not what was happening at the time of the problem.
Are you still interested in these logs?

**Reply by Nerivec** [reply]
Yes, `Adapter disconnected` is the hard fail of Z2M, it relies on Home Assistant (or whatever system you are using to monitor the Z2M instance) to restart Z2M from scratch. For EZSP that can happen for 3 reasons, too many watchdog failures, failure to send data to the serial port (after a couple of tries), or failure of the serial port. Since the whole stack relies on the adapter and Z2M to keep in sync, any of these requires a full restart of the stack. It previously restarted internally (without reaching the Z2M/HA level, but that lead to issues, so now it's a proper restart -like the other adapter types-).

I'd need the logs right before the crash indeed. See if there are any `error` (using search) in the logs you have, in which case, they might help.
Is the timespan between restarts pretty close each time? That could be your system suspending your USB port I'd guess...

`cat /sys/module/usbcore/parameters/autosuspend` should give you the value of the autosuspend (-1 is disabled), depending on your system, as always.

**Reply by merlinpimpim** [reply]
As expected, here is the link to the new issue: https://github.com/Koenkk/zigbee2mqtt/issues/21198

**Comment by nickb24** [comment]
The update broke the Stelpro HT402 Outdoor temperature command for some reason. I opened an issue here #21152 

Rolling back to 1.35.1 immediately makes the command work again.

**Comment by rrakso** [comment]
![image](https://github.com/Koenkk/zigbee2mqtt/assets/23143274/9e358cfa-0a25-4284-b389-e89d2b013517)
After performing the update, the add-on does not start 😕

**Reply by MrFaberinho** [reply]
u can click the refresh button

**Comment by arteck** [comment]
same problem with https://github.com/Koenkk/zigbee2mqtt/issues/21078

**Comment by weljajoh** [comment]
After upgrading to 1.35.2 the binding of my IKEA remote (E1524/E1810) to a Tradfri LED driver (ICPSHC24-30EU-IL-1) group was gone. It was now bound to something called 'default_bind_group'. Did a successfull rebind to the correct group through the GUI.

**Comment by MrFaberinho** [comment]
Anyone familiar with this RangeError?
Trying to connect EnOcean switch with Conbee III stick.

When pressing one of the four buttons I see this error in the debugger.

![Capture](https://github.com/Koenkk/zigbee2mqtt/assets/157371242/0d285231-6d29-4244-8928-f1199fab0da1)

**Reply by chris-1243** [reply]
It seems to be related to this issue [#12031](https://github.com/Koenkk/zigbee2mqtt/issues/12031)
